src/reports/models/aggregateReportManager.js

```diff
@@ -63,6 +66,23 @@ async function aggregateReport(report) {
     return reportInput;
 }
 
+async function getChaosExperiments(job_id) {
+    const chaosJobExperiments = await chaosExperimentsManager.getChaosJobExperimentsByJobId(job_id);
+    const uniqueExperimentIds = [...new Set(chaosJobExperiments.map(jobExperiment => jobExperiment.experiment_id))];
+    const chaosExperiments = await chaosExperimentsManager.getChaosExperimentsByIds(uniqueExperimentIds);
+    const mappedChaosJobExperiments = chaosJobExperiments.map((jobExperiment) => {
+        const chaosExperiment = chaosExperiments.find((experiment) => experiment.id === jobExperiment.experiment_id && jobExperiment.is_triggered);
```

Comment:
is it pissible that chaosExperiment will not be found? in this case you will get undefined and get unhandled execption on next line

Reply:
very edge case but added a condition